src/mpv_calendar/server.py

```diff
@@ -77,17 +77,19 @@ def _restart_service(service: str, user: bool, kill_filter: str) -> int:
     return result
 
 
-def _load_file_and_skip(item: str, last_playing_file: Path, default_image_fallback: bool) -> int:
+def _load_file_and_skip(
+    item: str, last_playing_file: Path, default_image_fallback: bool, queue: bool
+) -> int:
     """Construct and send the "loadfile" command over the socket."""
     if default_image_fallback:
         loadfile_command = (
-            f'echo \'{{ "command": ["loadfile", "{item}", "append-play"] }}\' '
+            f'echo \'{{ "command": ["loadfile", "{item}", "append-play"] }}'
+            f'\n{{ "command": ["loadfile", "{Constant.default_image}", "append-play"] }}\' '
             f"| socat - {Constant.mpv_socket}"
         )
     else:
         loadfile_command = (
-            f'echo \'{{ "command": ["loadfile", "{item}", "append-play"] }}'
-            f'\n{{ "command": ["loadfile", "{Constant.default_image}", "append-play"] }}\' '
+            f'echo \'{{ "command": ["loadfile", "{item}", "append-play"] }}\' '
             f"| socat - {Constant.mpv_socket}"
         )
```

Comment:
Implement the fallback to the default image based on the flag. Previously, the logic was wrong, but the flag wasn't ever changed.